samcli/lib/package/s3_uploader.py

```diff
@@ -85,7 +85,7 @@ def upload(self, file_name: str, remote_path: str) -> str:
 
         # Check if a file with same data exists
         if not self.force_upload and self.file_exists(remote_path):
-            LOG.debug("File with same data is already exists at %s. " "Skipping upload", remote_path)
+            LOG.info("File with same data is already exists at %s. " "Skipping upload", remote_path)
```

Comment:
Let's fix the message too: `A file with the same name already exists at %s`
- It seems we are checking the file path, not its content/data
- Remove the final `.`
- Fix the grammar

Reply:
Thanks @mgrandis ;
- The filename is generated md5 from its contents at line 140, so checking its name is basically checking its contents
- Done
- Done